src/media/css/tiles.styl

```diff
@@ -179,6 +179,20 @@ $greater-than-mobile = unquote('(min-width: 330px)');
     margin-bottom: 10px;
 }
 
+
+// Language packs
+.mkt-tile.langpack {
+    align-items: center;
+    display: flex;
```

Comment:
Voluntarily did not bother with the mixin because this is really a Firefox OS 2.2+ only feature, that no other browser/device will ever see, at least not in its current form.